recipes/cellannotation/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "point-annotator" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e05bb15d1a4a79a02222bdc51e753ca4ba4cf9a0f13d4ce0dd90ed4226727f0d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python      >=3
```

For consistency with most other conda-forge recipes, can you put a single space here and in the other dependencies?
```suggestion
    - python >=3
```